src/main/java/org/sonar/plugins/csharp/CSharpSensor.java

```diff
@@ -294,8 +307,7 @@ private void handleRoslynIssue(String repositoryKey, String ruleId, String uri,
     private final NoSonarFilter noSonarFilter;
     private final ResourcePerspectives perspectives;
 
-    public AnalysisResultImporter(Project project, SensorContext context, FileSystem fs, FileLinesContextFactory fileLinesContextFactory, NoSonarFilter noSonarFilter,
-      ResourcePerspectives perspectives) {
+    public AnalysisResultImporter(SensorContext context, FileSystem fs, FileLinesContextFactory fileLinesContextFactory, NoSonarFilter noSonarFilter, ResourcePerspectives perspectives) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 186 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00103)